src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+
+const dataAtributToNumber = (salary) => {
+  return salary.replace(/\D/g, '');
```

Comment:
This function `dataAtributToNumber` converts a string to a number by removing all non-digit characters. But it returns a string, not a number. In JavaScript, it's important to keep data types consistent. You should convert the result to a number using the `Number()` function.